frontend/src/components/MealsList.jsx

```diff
@@ -43,11 +43,12 @@ function MealsList() {
   return (
     <>
       <Search search={search} setSearch={setSearch} searchHandler={searchHandler} />
+      
       <div className="meal-list">
         {loading ? (
           <p>Loading...</p>
         ) : meals.data.length > 0 ? (
-          meals.data.map((meal) => <MealItem key={meal.meal_id} meal={meal} />)
+          meals.data.map((meal,index) => <MealItem key={index} meal={meal} />)
```

Comment:
its not considered a good practice to use index for `key`, please change to id ( you are using it in the `MealItem` already)

Reply:
done!